internal/plans/deferring/deferred.go

```diff
@@ -201,6 +213,8 @@ func (d *Deferred) HaveAnyDeferrals() bool {
 // IsResourceInstanceDeferred returns true if the receiver knows some reason
 // why the resource instance with the given address should have its planned
 // action deferred for a future plan/apply round.
+// TODO NF: This is dead code, and we always ignore the instructions below
+// to use it before ShouldDeferResourceInstanceChanges.
 func (d *Deferred) IsResourceInstanceDeferred(addr addrs.AbsResourceInstance) bool {
```

Comment:
Is there any reason to keep this around if nothing is referencing it? Can you delete it now?